### PR TITLE
Replace the main cover picture

### PR DESCRIPTION
**Pull request by hroncok**
- it has better colors
 - it is free as in freedom

### CODE REVIEW

File: pyconcz/static_src/scss/pyconcz19/_home.scss [hunk 1]
```diff
@@ -12,9 +12,9 @@
 .pc-main-lead {
     position: relative;
     background-color: $black;
-    background-image: url(/2019/static/img/ostrava/20150413_zer_hlubina_044.jpg);
+    background-image: url(/2019/static/img/ostrava/vitkovice.jpg);
     background-repeat: no-repeat;
-    background-position: 50% 25%;
+    background-position: 50% 75%;
```
Comment (hroncok) [inline pyconcz/static_src/scss/pyconcz19/_home.scss]: This needs a rebuild, not sure how to do it.

Comment (frenzymadness) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: Description in README doesn't help?

Comment (hroncok) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: I'm not doing `sudo npm install gulp-cli -g` ever. Sorry.

Comment (hroncok) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: But I haven't read the README first, sorry about that. Now I know how to do it, but I don't want to :laughing:  Will try to search for gulp in the RPM repos.

Comment (encukou) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: `npm install gulp-cli` installs locally, and hides the binary in `./node_modules/.bin/gulp`.

File: pyconcz/templates/default/pages/minimal_landing.html [hunk 2]
```diff
@@ -14,7 +14,7 @@ <h1>PyCon CZ 2019</h1>
                 <div>Ostrava</div>
                 <div>June 14–16</div>
             </div>
-            <div class="pc-image-copyright">Photo © <a href="https://www.zerzon.cz/">Jiří Zerzoň</a></div>
+            <div class="pc-image-copyright">Photo 🄯 <a href="https://www.flickr.com/photos/kubajs/20958638705/">Jakub Souček</a></div>
```
Comment (frenzymadness) [inline pyconcz/templates/default/pages/minimal_landing.html]: What does this square character mean?

Comment (hroncok) [reply pyconcz/templates/default/pages/minimal_landing.html]: I see copyleft symbol https://www.fileformat.info/info/unicode/char/1f12f/index.htm

File: pyconcz/static_src/scss/pyconcz19/_home.scss (hunk 1 above)
Comment (frenzymadness) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: I manage to do that in [Docker container](https://hub.docker.com/r/akoehnlein/surf). You just need to start a container with shared folder via `-v` and then run `npm install` and `npm run build`.

Do you want to try it or should I send a PR on your fork?

Comment (hroncok) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: feel free to push directly to my fork.

File: pyconcz/templates/default/pages/minimal_landing.html (hunk 2 above)
Comment (encukou) [reply pyconcz/templates/default/pages/minimal_landing.html]: U+1F12F: COPYLEFT SYMBOL

But the license is CC BY 2.0, not copyleft. I'd just say "by".

Comment (hroncok) [reply pyconcz/templates/default/pages/minimal_landing.html]: right.

Comment (hroncok) [reply pyconcz/templates/default/pages/minimal_landing.html]: the CC unicode symbol is probably still not approved :(

**Comment by frenzymadness** [comment]
Adding screenshot

image

File: pyconcz/static_src/scss/pyconcz19/_home.scss (hunk 1 above)
Comment (benabraham) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: I will update npm (frontend) stack ASAP to remove any global installation.

Comment (benabraham) [reply pyconcz/static_src/scss/pyconcz19/_home.scss]: #28

**Comment by frenzymadness** [comment]
I did a squash and fixed conflicts caused by another PR. Looks good to me.

**Pull request merged**